consensus/config/src/parameters.rs

```diff
@@ -16,18 +17,27 @@ pub struct Parameters {
     /// Time to wait for parent round leader before sealing a block.
     #[serde(default = "Parameters::default_leader_timeout")]
     pub leader_timeout: Duration,
+
+    /// The database path
+    #[serde(default = "Parameters::default_db_path")]
+    pub db_path: PathBuf,
 }
 
 impl Parameters {
     pub fn default_leader_timeout() -> Duration {
         Duration::from_millis(250)
     }
+
+    pub fn default_db_path() -> PathBuf {
+        PathBuf::default()
```

Comment:
I've explicitly chosen to use the `PathBuf::default()` here as any initialise of storage with that path will end up with a failure. This is done for a few reasons:
* we don't accidentally bootstrap a node with some default path that creates unwanted directories etc
* an alternative was to use a tempdir , but this would make things more insecure in the sense that wecould accidentally boostrap a node with a temporary dir and lose history after restart
* wanted somehow to force the user to set a path but still make the `default` behaviour here work

Reply:
@arun-koshy good point, we could do `Option` as well, I've avoided so we don't end up with unset values but probably here makes more sense.

@mwtian the epoch number is currently appended by the caller (mysticet manager) in same fashion that is done for narwhal, so we probably don't need to do that here again.